resource/api/upload.go

```diff
@@ -18,8 +18,8 @@ import (
 
 // UploadRequest defines a single upload request.
 type UploadRequest struct {
-	// Service is the application ID.
-	Service string
+	// Application is the application ID.
+	Application string
```

Comment:
I think this renaming of property also means you'd need to bump facades that use/return this struct.

In fact, I wonder if we need to "deprecate" it - still carry around "Service" but keep it empty and use "Application" everywhere that matters.

Reply:
This is also not a struct send over the wire. It's used internally in the api and apiserver layers to encapsulate data - the content comes from unmarshalling a http request.